esp/scm/ws_packageprocess.ecm

```diff
@@ -288,7 +288,7 @@ ESPresponse [exceptions_inline] GetPartFromPackageMapResponse
     string Content;
 };
 
-ESPservice [requiredaccess("NONE"), version("1.02"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsPackageProcess
+ESPservice [autofeatureauth("NONE"), version("1.02"), exceptions_inline("./smc_xslt/exceptions.xslt")] WsPackageProcess
 {
```

Comment:
@rpastrana If you add one, will it default to allow, or will operations will have to manually update ldap for any updated environment?  If the latter are you going to coordinate with operations to enable it before they upgrade?